### PR TITLE
Added missing doc strings, #3413

### PR DESCRIPTION
**Pull request by endo64**
Added missing doc strings, #3413

```
<<                   : Shift bits to the left
>>                   : Shift bits to the right
>>>                  : Shift bits to the right (unsigned)
_read-input          : Internal Use Only
_set-buffer-history  : Internal Use Only
as-color             : Combine R, G and B values into a tuple
as-ipv4              : Combine a, b, c and d values into a tuple
as-rgba              : Combine a, b, c and d values into a tuple
ask                  : Prompt the user for input
bind                 : Bind words to a context; returns rebound words
charset              : Shortcut for `make bitset!`
comment              : Consume but don't evaluate the next value
common-substr        : Internal Use Only
construct            : Makes a new object from an unevaluated spec; standard logic words are evaluated
context              : Makes a new object from an evaluated spec
create-dir           : Create the given directory
debug-info?          : Internal use only
dir                  : Display a single column directory listing, for the current dir if none is given
dir?                 : Returns TRUE if the value looks like a directory spec
do-actor             : Internal Use Only
do-file              : Internal Use Only
do-safe              : Internal Use Only
eval-set-path        : Internal Use Only
event?               : Returns true if the value is this type
exists?              : Returns TRUE if the file exists
find-flag?           : Checks a flag in a face object
get-current-dir      : Returns the platform’s current directory for the process
halt                 : Stops evaluation and returns to the input prompt
input                : Wait for console user input
last-lf?             : Internal Use Only
link-sub-to-parent   : Internal Use Only
link-tabs-to-parent  : Internal Use Only
ll                   : Display a single column directory listing, for the current dir if none is given
ls                   : Display a single column directory listing, for the current dir if none is given
make-face            : Make a face from a given style name or example face
normalize-dir        : Return the contents of the system clipboard
object               : Makes a new object from an evaluated spec
on-face-deep-change* : Internal use only
on-parse-event       : Standard parse/trace callback used by PARSE-TRACE
os-info              : Returns detailed operating system version information
parse                : Process a series using dialected grammar rules
pwd                  : Displays the active directory path (Print Working Dir)
quote                : Return but don't evaluate the next value
read-clipboard       : Return the contents of the system clipboard
red-complete-file    : Internal Use Only
red-complete-input   : Internal Use Only
red-complete-path    : Internal Use Only
routine              : Defines a function with a given Red spec and Red/System body
set-current-dir      : Sets the platform’s current process directory
set-flag             : Sets a flag in a face object
shift-left           : Shift bits to the left
shift-logical        : Shift bits to the right (unsigned)
shift-right          : Shift bits to the right
to-red-file          : Converts a local system file path to a Red file path
update-font-faces    : Internal Use Only
write-clipboard      : Write content to the system clipboard
write-stdout         : Write data to STDOUT
```

### CODE REVIEW

**Comment by DideC** [comment]
`dir` and `ls` produce a "...**multi columns** directory listing...", while `ll` produce a "...single column..." one !!!

**Comment by DideC** [comment]
In this list, the spec for `normalize-dir` is also wrong.

File: environment/functions.red [hunk 1]
```diff
@@ -564,9 +568,10 @@ to-red-file: func [
 	dst
 ]
 
-dir?: func [file [file! url!]][#"/" = last file]
+dir?: func ["Returns TRUE if the value looks like a directory spec" file [file! url!]][#"/" = last file]
 
 normalize-dir: function [
+	"Return the contents of the system clipboard"
```
Comment (DideC) [inline environment/functions.red]: Not really what it does, isn't it ?

File: environment/console/engine.red [hunk 2]
```diff
@@ -310,10 +310,10 @@ expand: func [
 	probe expand-directives/clean blk
 ]
 
-ls:		func ['dir [any-type!]][list-dir :dir]
-ll:		func ['dir [any-type!]][list-dir/col :dir 1]
-pwd:	does [prin mold system/options/path]
-halt:	does [throw/name 'halt-request 'console]
+ls:		func ["Display a single column directory listing, for the current dir if none is given" 'dir [any-type!]][list-dir :dir]
```
Comment (DideC) [inline environment/console/engine.red]: this is a "multi columns directory listing" !

**Comment by greggirwin** [comment]
Thanks for reviewing @DideC ! Do you want to make a PR to fix them, or have @Endo do it?

**Comment by DideC** [comment]
I let Endo do it ;-)
Don't know what to put for `normalize-dir`.

**Comment by meijeru** [comment]
@DideC @endo64 what `normalize-dir` does:
`normalize-dir` will invoke `clean-path` and prefix the current working directory if the file path indicated by its argument (`word!` `path!` or `file!`) is _relative_, and add a trailing `/` if necessary
`clean-path` will normalize its `file!` argument to an absolute path where `.` and `..` are eliminated

**Comment by greggirwin** [comment]
It was just a bad entry. Endo and I developed the list, and that one just ended up in the wrong slot. Normalize-dir will say "Returns an absolute directory spec" when fixed.

**Comment by greggirwin** [comment]
`Ls` will be "Display a directory listing, for the current dir if none is given"

**Comment by endo64** [comment]
@DideC Thanks for catching! 
@greggirwin I fixed them and made a PR #3419

**Comment by DideC** [comment]
@endo64 @greggirwin  thanks for the averall fix and this correction.

**Pull request merged**